src/test/scala/org/broadinstitute/dsde/firecloud/service/NihServiceUnitSpec.scala

```diff
@@ -272,6 +295,33 @@ class NihServiceUnitSpec extends AnyFlatSpec with Matchers with BeforeAndAfterEa
     verifyTargetGroupSynced()
   }
 
+  it should "sync all users by including groups found with consentGroup + phsId" in {
+    when(ecmDao.getActiveLinkedEraAccounts(ArgumentMatchers.eq(UserInfo(adminAccessToken, ""))))
+      .thenReturn(Future.successful(Seq(userDbGapListLinkedAccount)))
+    when(thurloeDao.getAllUserValuesForKey(any[String])).thenReturn(Future.successful(Map.empty))
+
+    val emailsToSync = Set(userDbGapList.email, userDbGapGroup.email)
```

Comment:
I don't understand what these 2 emails are. They seem to be users but that is not what I was expecting the test to be looking for. The test should be regardless of users. Even when there are no linked users, we should verify that the configured dbgap group gets added

Reply:
Ah, it needs to be the email of the *group*?   Apparently that went over my head and I was think it was *user* emails.  Should be mostly the same, just need to update some names in the test.